### ISSUE TITLE
TODO Lists

### ISSUE THREAD

**Issue by bibbca** [kaushalmodi/writing-org#1]
Hi,  I'm new to both this fork and the josephernest/writing.  In my reading of the documentation of org mode, it mentions TODO's. Is there TODO's in this implementation of org mode?

**Comment by kaushalmodi** [comment]
No, there isn't. But how would you visualize the TODO's to be implemented in this project?

I do not plan this project to be a rewrite of Org in JS (also I don't know JS.. Just enough to make the kind of changes I have done so far). 

This project is to help break the barrier for people new to Org, and willing to use it as Markdown replacement. The Org TODO's are more than visual tags inside Org (See the [Org manual sections on TODO][1]). 

But for this project, I can limit it to visual-only representation .. Have the TODO keywords show in red, and DONE keywords show in green. 

[1]: http://orgmode.org/manual/TODO-basics.html

**Comment by bibbca** [comment]
I think that the keyword colors would be a great help.  Although ideally I would hope for a checkbox to at least be displayed (it doesn't need to have a function, it could just be a box).
